base/Dockerfile.prepare.erb

```
@@ -1,10 +1,15 @@
 COPY base/bin/prepare /tmp/devon_rex_work/
 RUN /tmp/devon_rex_work/prepare
 
+# Install Git and verify the specified version of Git is installed
+COPY --from=git-builder /opt/git /
```

[suggestion] How about copying to `/opt/git` as follows?

```suggestion
COPY --from=git-builder /opt/git /opt/git/
```

---

If I changed it like that, I have to reconfigure `PATH` with `PATH=/opt/git/usr/local/bin:$PATH`. I think git(1) should be located in `/usr/local/bin`.

---

Oh, I got it. 👍